src/placement/pl_map.c

```diff
@@ -583,10 +583,7 @@ pl_select_leader(daos_obj_id_t oid, uint32_t grp_idx, uint32_t grp_size,
 		return shard->po_shard == -1 ? -DER_IO : shard->po_shard;
 	}
 
-	replicas = oc_attr->u.rp.r_num;
-	if (replicas == DAOS_OBJ_REPL_MAX)
-		replicas = grp_size;
-
+	replicas = grp_size;
```

Comment:
Then we can use "grp_size" directly and remove "replicas".

Reply:
ok, let me refresh it.